### DISCUSSION TITLE
NCBI Gene Update

### DISCUSSION THREAD

**Discussion by kokyriakidis** [KarchinLab/open-cravat#247] [Usability & Feature Requests]
Hi!

Is there an easy way to update NCBI Gene module?

Thanks!

**Comment by kmoad** [comment]
Yeah, looks like the data is old. We'll update it. Thanks for pointing this out.

We're working on automatic data updates for annotators like ncbigene that regularly get new data.

**Comment by mstucky** [comment]
A new version of the NCBI Gene module has been published to the OC Store.  It's now up on run.opencravat.org.  The new version number is 2022.08.30.
